int64 A_target
float64 A_objx
float64 A_objy

int64[] bbox_size
float64[] B_target_x
float64[] B_target_y

string signname